omod/src/main/java/org/openmrs/module/pihapps/rest/LabOrderConfigRestController.java

```diff
@@ -1,90 +0,0 @@
-package org.openmrs.module.pihapps.rest;
-
-import org.apache.commons.lang.StringUtils;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
-import org.openmrs.CareSetting;
-import org.openmrs.EncounterRole;
-import org.openmrs.EncounterType;
-import org.openmrs.api.EncounterService;
-import org.openmrs.api.OrderService;
-import org.openmrs.module.webservices.rest.SimpleObject;
-import org.openmrs.module.webservices.rest.web.response.ResponseException;
-import org.openmrs.util.ConfigUtil;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.ResponseBody;
-
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-/**
- * Provides configuration and other endpoints for lab order entry
- */
-@Controller
-public class LabOrderConfigRestController {
```

Comment:
Deleted this class in favor of a broader pihAppsConfig REST controller, as lab order config can be retrieved via that as needed.